tests/meta/report/conftest.py

```diff
@@ -32,41 +36,48 @@ def report_api_mip_dna(cg_context: CGConfig, lims_samples, report_store) -> MipD
 
 
 @pytest.fixture(scope="function", name="report_api_balsamic")
-def report_api_balsamic(cg_context: CGConfig, lims_samples, report_store) -> BalsamicReportAPI:
+def report_api_balsamic(
+    cg_context: CGConfig, lims_samples: List[dict], report_store: Store
+) -> BalsamicReportAPI:
     """BALSAMIC ReportAPI fixture."""
-
     cg_context.meta_apis["analysis_api"] = MockBalsamicAnalysis(cg_context)
     cg_context.status_db_ = report_store
     cg_context.lims_api_ = MockLimsAPI(cg_context, lims_samples)
     cg_context.scout_api_ = MockScoutApi(cg_context)
     return BalsamicReportAPI(cg_context, cg_context.meta_apis["analysis_api"])
 
 
+@pytest.fixture(scope="function", name="report_api_rnafusion")
+def report_api_rnafusion(
+    rnafusion_context: CGConfig, lims_samples: List[dict]
+) -> RnafusionReportAPI:
+    """Rnafusion report API fixture."""
+    rnafusion_context.lims_api_ = MockLimsAPI(rnafusion_context, lims_samples)
+    rnafusion_context.scout_api_ = MockScoutApi(rnafusion_context)
+    return RnafusionReportAPI(rnafusion_context, rnafusion_context.meta_apis["analysis_api"])
```

Comment:
Use named args

Reply:
For some reason this is an invalid typehint usage 🤔